### ISSUE TITLE
Question about editing INI

### ISSUE THREAD

**Issue by MarcusOSG** [1dot13/gamedir#100]
I edited some values in INI, but I have a question about columns that can only be changed between TRUE and FALSE, how exactly can I edit them?

Random Ex: 

   <Property name="ALLOW_EXPLOSIVE_ATTACHMENTS" datatype="boolean" minvalue="0" maxvalue="0" defaultvalue="0" interval="0">
       <Description_ENG>Default: FALSE
(Text explaining what it does).</Description_ENG>

My question is, how do I activate or deactivate it? In "<Description" or "</Description"? Or changing the values?  Some other texts do not have "Default:"

Ex: 

      <Property name="AI_EXTRA_SUPPRESSION" datatype="boolean" minvalue="0" [...]
          <Description_ENG>allow AI to shoot more often for suppression in O THE by taking minium CTH=1 for autofire capable weapons</Description_ENG>

So what should I actually put after "</Description_ENG"? Or should I create one line below property name like que first one? Or something else?

I really need a quick and easy to understand guidance, I'm a little slow xD, I'm just starting to modding games, my bad if it's too dumb/too easy. If you can give me a example (without shortening) of how put a full setting in false and true I would really appreciate if this applies to all others.

Also I'm in middle game and I really didn't want to go back to the beginning, but I would like to know if applying the following settings in mid-game they will work or not:

Allow explosive att

Chance values of CHT like in head and unseen targets
 
Allow vehicles to run over people and against obstacles (walls, tress, etc)

Allow enemies vehicles/patrols vehicles and allow them to move on tactical map (idk if it's by default or not but in my current game/settings I cannot run over enemies at least, I always have to pass by or change my route because of a single enemy in the middle of the road, is it too unstable?)

AND MAINLY In the current settings, how do I change the progression of items as a whole?  I think items/weapons, etc. advance too quickly in a short period of time, is there any way I can slow this down or at least "pause" this process?

Sorry for long text but I really want to remove all my doubts.

Thanks

**Comment by MarcusOSG** [comment]
I just noticed, I have to access the game's directory: JA2, then go DATA 1.13? To edit the settings?

**Comment by Asdow** [comment]
Your example makes me think you're trying to edit INIEditorJA2Options.xml, which is used to tell the INI editor.exe what kind of options are available.


What you want to edit is indeed JA2_Options.ini file that can be found in Data-1.13 subfolder

**Comment by MarcusOSG** [comment]
> Your example makes me think you're trying to edit INIEditorJA2Options.xml, which is used to tell the INI editor.exe what kind of options are available.
> 
> What you want to edit is indeed JA2_Options.ini file that can be found in Data-1.13 subfolder

Yes, I searched a little more and ended up finding it, my only questions is how do I mark items for the militia not to use? the files it's saying that I should press TAB, I do but nothing happens And is there any way in INI to slow down or pause the progression of items? And How do I run over people using vehicles? Do I need to hold shift before moving? I already actived it on INI.
